website/for_teachers.py

```diff
@@ -1140,9 +1199,13 @@ def create_adventure(self, user):
             "date": utils.timems(),
             "creator": user["username"],
             "name": name,
+            "classes": class_id,
             "level": 1,
             "content": "",
             "language": g.lang,
         }
         self.db.store_adventure(adventure)
+        if class_id != '0':
```

Comment:
instead of this, either
- pass the `class_id` as a parameter and then simply check if it's passed to this endpoint
- or make it optional and then simply check `if class_id: ..`

this way you don't have to pass do:
`hx-post="/for-teachers/create-adventure/0"`

Reply:
but rather `hx-post="/for-teachers/create-adventure"` if no class id needed, or`hx-post="/for-teachers/create-adventure?class_id={{class.id}}"` or `hx-post="/for-teachers/create-adventure/{{class_id}}"` if needed!